xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOMX.cpp

```diff
@@ -27,6 +27,7 @@
 #include "utils/GLUtils.h"
 #include "settings/MediaSettings.h"
 #include "windowing/WindowingFactory.h"
+#include "DVDCodecs/Video/DVDVideoCodecIMX.h"
```

Comment:
OpenMax should not have anything to do with the IMX decoder / render which is for IMX6 only.